tutorial/Lab4/README.md

```diff
@@ -6,7 +6,7 @@ A means of sharing is a chart repository, which is a location where packaged cha
 
 # Using charts from a public repository
 
-Helm charts can be available on a remote repository or in a local environment/repository. The remote repositories can be public like [Helm Charts](https://github.com/helm/charts) or [IBM Helm Charts](https://github.com/IBM/charts), or hosted repositories like on Google Cloud Storage or GitHub. Refer to [Helm Chart Repository Guide](https://v2.helm.sh/docs/developing_charts/#the-chart-repository-guide) for more details. 
+Helm charts can be available on a remote repository or in a local environment/repository. The remote repositories can be public like [Bitnami Charts](https://github.com/bitnami/charts) or [IBM Helm Charts](https://github.com/IBM/charts), or hosted repositories like on Google Cloud Storage or GitHub. Refer to [Helm Chart Repository Guide](https://helm.sh/docs/topics/chart_repository/) for more details. 
```

Comment:
Why change /helm/charts to /bitnami/charts?

Reply:
This is because the helm chart repository is going out of support in parallel with Helm v2 support.    Charts will ne need top be hosted by the chart maintainer. Details on this: https://github.com/helm/charts#status-of-the-project